docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
 
 You will need credentials to log in and upload metadata. 
 
-LifeCycle, ATHLETE and LongITools use [*data-catalogue-staging*](https://data-catalogue-staging.molgeniscloud.org).  
+Cohorts in projects such as ATHLETE, IPEC and LongITools use [*data-catalogue-staging*](https://data-catalogue-staging.molgeniscloud.org).  
 ConcePTION uses [*conception-acc*](https://conception-acc.molgeniscloud.org).
 
 When you log in, you will be able to see at least the following databases:
 
-- <b> DataCatalogue</b>: The catalogue data, in which you can search for target variables to map to.
+- <b>DataCatalogue</b>: The catalogue data, in which you can search for target variables to map to.
 - <b>CatalogueOntologies</b>: This database contains the look-up list that you need for filling out some columns in the
   templates, e.g. format or unit. If you need to add anything to these look-up lists, contact us
   at [molgenis-support](mailto:molgenis-support@umcg.nl).
-- <b>SharedStaging</b>: A communal staging area in which e.g. Contacts and Institutions are added and edited.
-- <b>Your own database</b>: use this to upload the templates once you have filled them out.
+- <b>Your own database </b>: (here: testCohort) Use this to fill out cohort rich metadata and to upload the templates once you have filled them out.
 
 ![MOLGENIS databases](../img/cat_databases.png)
 
 <sup>*Figure 1. Databases in the Data Catalogue staging area.*</sup>
 
 ### Fill out cohort rich metadata
 
-Open your staging area, navigate to 'Tables' and open the table 'Cohorts'. Your cohort pid and name are already 
+Open your staging area, navigate to 'Tables' and open the table 'Cohorts'. Your cohort id and name are already 
 filled out. Click on the pencil sign next to this entry to start editing your cohort rich metadata by filling out 
-the form.
+the form. 'Subcohorts' and 'Collection events' should also be filled out through this route. You can fill them out 
+in subsections inside the 'Cohorts' form.
 
 ### Define metadata of cohorts or data sources
 
 This section explains how to submit the 'source variables' + 'mappings from source variables to target variables' into
 the Data Catalogue. Expected users of this 'how to' are data managers within the institutions with access to cohorts or
```

Comment:
rename institutions to organisation?

Reply:
i think deleting 'within the institutions' from the sentence will be better